public/data/kapoorsaumitra.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Saumitra Kapoor",
+  "type": "personal",
+  "bio": "I am a software engineer and a writer. I write about technology, startups, and life.",
+  "avatar": "https://pbs.twimg.com/profile_images/1557598338803200000/E7bmIo-L_400x400.jpg",
```

Comment:
Code looks good to me, but it would be great if you use the standard link for the profile avatar! 🛠

```suggestion
  "avatar": "https://github.com/kapoorsaumitra.png",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient